app/apps.json

```diff
@@ -205,5 +205,32 @@
       "type": "addon",
       "url": "https://github.com/xeonchen/fxos-line-fixer",
       "revision": "4c93314c2fa985aa41316f9423f06090dec059d4"
+    },
+    "https://lmorchard.github.io/fxos-addon-messages-bigger-send-button/manifest.webapp": {
```

Comment:
Thanks! I understand it can be a lot of work, but it seems to improve the situation a lot so I wanted to suggest it.